mantidproject/mantid
# Restore help button to Manage User Directories

**Description of work.**
The Manage User Directories window is no longer modal, which means that the help menu can be opened from it without causing problems. It is not possible to have more than one Manage User Directories window open at a time.

**To test:**
1. File -> Manage User Directories. Check that there is a help button in the bottom-left corner and it works correctly.
2. Attempt to open another Manage User Directories window and make sure you can't. This should also work for opening the window in an interface, such as Muon -> Elemental Analysis.

Fixes #26429 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
